cicd/build.sh

```diff
@@ -16,21 +16,37 @@ set -e
 
 source ${CICD_ROOT}/_common_container_logic.sh
 
+is_pr_or_mr_build() {
+    [ -n "$ghprbPullId" ] || [ -n "$gitlabMergeRequestId" ]
+}
+
+get_expiry_label_parameter() {
+    echo "--label quay.expires-after=${QUAY_EXPIRE_TIME}"
+}
+
+is_rhel7_host() {
+
+    local RELEASE_FILE='/etc/redhat-release'
+
+    [ -f "$RELEASE_FILE" ] && grep -q -i "release 7" "$RELEASE_FILE"
+}
+
 function build {
+
+    local IMAGE_TAG_LATEST=''
+
     if [ ! -f "$APP_ROOT/$DOCKERFILE" ]; then
         echo "ERROR: No $DOCKERFILE found"
         exit 1
     fi
 
-    # if this is a PR, set the tag to expire in 3 days and set arg 'TEST_IMAGE=true'
-    if [ ! -z "$ghprbPullId" ] || [ ! -z "$gitlabMergeRequestIid" ]; then
-        echo "LABEL quay.expires-after=${QUAY_EXPIRE_TIME}" >> $APP_ROOT/$DOCKERFILE
```

Comment:
@bsquizz  the reason for the PR title being "set as build parameter" is because I removed this particular line(old line 27), that edited the app's Dockerfile, and instead used is as an option to the `docker` / `podman` *build* command, just a few lines after it in (the new line 44), where I add it to the `CMD_OPTS` variable. So the expected build command should look like:

```
podman/docker build 
 -t ${IMAGE}:${IMAGE_TAG} 
 -t ${IMAGE}:${IMAGE_TAG_LATEST}
 --label quay.expires-after=${QUAY_EXPIRE_TIME}
 --build-arg TEST_IMAGE=true -t 
```

not to be confused with a *build-arg* . I have renamed the PR to properly reflect this change, hope it's clearer then . In other words, my intention here is to avoid to touch the app's Dockerfile and instead set the label as a parameter to `build`. Hope that makes sense.

Reply:
I see! Clearly I missed that :D